docs/user-event/install.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: install
+title: Installation
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+<Tabs defaultValue="npm" values={[
+    { label: 'npm', value: 'npm' },
+    { label: 'Yarn', value: 'yarn' },
+]}>
+<TabItem value="npm">
+
+```sh
+npm install --save-dev @testing-library/user-event@^14.0.0-beta
```

Comment:
Since we're already tagging this release as beta, don't we want to use that version tag?

```suggestion
npm install --save-dev @testing-library/user-event@beta
```

Reply:
:shrug: I don't know. Doesn't make a difference.
IMHO in case of doubt, I'd go with the more explicit one as it once again highlights the major version bump. We probably won't use the `beta` channel for a while after is is released.